modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -681,7 +681,7 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       encryptionKeys: [bitgoGpgKey],
     })) as string;
     const userGpgKey = await generateGPGKeyPair('secp256k1');
-    const privateShareProof = await createShareProof(userGpgKey.privateKey, signingKey.nShares[bitgoIndex].u, 'ecdsa');
+    const privateShareProof = await createShareProof(userGpgKey.privateKey, signingKey.nShares[bitgoIndex].y, 'ecdsa');
```

Comment:
This is not right, actually. You should still use the `u` value there and add another line
```
    const publicShare = signingKey.nShares[bitgoIndex].y + signingKey.nShares[bitgoIndex].chaincode; // add chaincode to be consistent with other publicShare
```
And add the `publicShare` in as another field with `vssProof` and `privateShareProof`.